### PR TITLE
[C-859] Omit `is_verified` from metadata that gets stored in user cache from fetchCID

### PR DESCRIPTION
**Pull request by sliptype**
### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sk-c-859-dont-read-verified-from-cn

File: packages/web/src/pages/profile-page/sagas.js [hunk 1]
```diff
@@ -85,7 +85,10 @@ function* fetchProfileCustomizedCollectibles(user) {
         cacheActions.update(Kind.USERS, [
           {
             id: user.user_id,
-            metadata: { ...metadata, collectiblesOrderUnset: true }
+            metadata: {
```
Comment (raymondjacobson) [inline packages/web/src/pages/profile-page/sagas.js:88]: also on line 76

Comment (sliptype) [reply packages/web/src/pages/profile-page/sagas.js:88]: oops, fixed

**Review by raymondjacobson — APPROVED** [review]
unnecessary import, but lgtm now!

**Pull request merged**